Dockerfile

```diff
@@ -18,13 +18,15 @@ COPY controllers/ controllers/
 COPY pkg/ pkg/
 
 # Build
+ARG LOCAL_BUNDLE=odh-manifests.tar.gz
```

Comment:
You will need to move this line after the `FROM` line below.

The [scope](https://docs.docker.com/engine/reference/builder/#scope) of this ARG is restricted to the builder image so the `COPY $LOCAL_BUNDLE` argument below will just copy the entire contents of the repo into `/opt/manifests`

Reply:
```suggestion
```